components/clp-py-utils/clp_py_utils/initialize-clp-metadata-db.py

```diff
@@ -5,6 +5,8 @@
 from contextlib import closing
 from pathlib import Path
 
+from sql_adapter import SQL_Adapter
+
```

Comment:
just curious. did you revert this change because it's not necessary for this PR?

Reply:
Oui